polly/htmx_endpoints.py

```
@@ -2265,73 +2294,81 @@ async def get_stats_htmx(
+                    try:
+                        from sqlalchemy import func as sa_func
+                        vote_count = (
+                            await db.execute(
+                                select(sa_func.count())
+                                .select_from(Vote)
+                                .where(Vote.poll_id == TypeSafeColumn.get_int(poll, "id"))
+                            )
+                        ).scalar_one()
+                        if isinstance(vote_count, int):
+                            total_votes += vote_count
+                    except Exception as fallback_e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)